### PR TITLE
Add libretro netplay support

### PR DESCRIPTION
**Pull request by substring**
- adds a new commandline switch : -host/-client
- expects 4 new recalbox.conf vars :
  - netplay.nick
  - netplay.frames
  - netplay.server.ip
  - netplay.server.port

Needs a nice ES UI now

### CODE REVIEW

File: configgen/emulatorlauncher.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def main(args):
     if systemName in emulators:
         system = emulators[systemName]
         systemManager = ConfigManager()
-        systemManager.configure(system, args.emulator, args.core, args.ratio)
+        systemManager.configure(system, args.emulator, args.core, args.ratio, args.netplay)
```
Comment (digitalLumberjack) [inline configgen/emulatorlauncher.py]: Why do we need netplay in commandline args ? Can it be overriden for a game ?

**Comment by nadenislamarre** [comment]
nice.
It would be nice to add a switch for client spectator mode, no ?

**Comment by substring** [comment]
You have to know how a rom is launched : host or client (or spectator)
That is something that is not constant, wheras the nick, the frames or the server ip have less chances to change.
The frames depends on the ping between machines
The port, most of times, should be constant
The server ip will change, but that only mayters for client(and spectator)
The idea behin is to add a dew options ins ES:
- start a rom as server or client (or spectator)
- make the server ip, frames and nick configirable inside ES

I don't mind changing anything, just make proposals.

**Comment by substring** [comment]
I have some other bigger ideas for spectator mode, but that is in a distant future ^^

**Comment by pablomacaluso** [comment]
Its working?

**Pull request merged**